packages/google_maps_flutter/pubspec.yaml

```diff
@@ -2,7 +2,7 @@ name: google_maps_flutter
 description: A Flutter plugin for integrating Google Maps in iOS and Android applications.
 author: Flutter Team <flutter-dev@googlegroups.com>
 homepage: https://github.com/flutter/plugins/tree/master/packages/google_maps_flutter
-version: 0.5.20
+version: 0.5.20+1
```

Comment:
0.5.21

Reply:
It's 0.5.21 to me

Reply:
hmm. I saw it shows 0.5.20+1. Let me try this:
```suggestion
version: 0.5.21
```